### ISSUE TITLE
Needs support for the World Imagery (Clarity) basemap

### ISSUE THREAD

**Issue by najoco** [Esri/esri-leaflet#1041]
A new basemap named World Imagery (Clarity) is available.  Adding this as an additional basemap would be beneficial.

See this blog post:
https://blogs.esri.com/esri/arcgis/2017/07/06/world-imagery-view-clarity/

Further info on the basemap:
https://www.arcgis.com/home/item.html?id=ab399b847323487dba26809bf11ea91a

**Comment by jgravois** [comment]
i'd definitely be interested in adding `"ImageryClarity"` as a new basemap string. are you interested in submitting it via a pull request? if so, check out #758 to see an example of what is required.

in the meantime folks can add the layer manually.

```js
L.esri.tiledMapLayer({
    url: "https://clarity.maptiles.arcgis.com/arcgis/rest/services/World_Imagery/MapServer",
    attribution: 'DigitalGlobe, GeoEye, Earthstar Geographics, CNES/Airbus DS, USDA, USGS, AeroGRID, IGN, and the GIS User Community'
  }).addTo(map);
```

**Comment by jgravois** [comment]
closed via #1047
